unit_tests/testSpeciesElements/testSpeciesElements.f90

```diff
@@ -38,36 +37,36 @@ PROGRAM testSpeciesElements
 !-------------------------------------------------------------------------------
    SUBROUTINE testSpeciesElementsBase()
       TYPE(StringType) :: testWord
-      REAL(SRK), DIMENSION(0:118) :: testArray
+      REAL(SRK), DIMENSION(1:119) :: testArray
 
       ! Test H2O
       testWord = 'H2O'
-      testArray = mySpec%getElementArray(testWord)
+      testArray = getElementArray(testWord)
       ASSERT_EQ(testArray(myEI%getAtomicNumber('H')),2.0,'')
       ASSERT_EQ(testArray(myEI%getAtomicNumber('O')),1.0,'')
 
       ! Test C2H6O
       testWord = 'C2H6O'
-      testArray = mySpec%getElementArray(testWord)
+      testArray = getElementArray(testWord)
       ASSERT_EQ(testArray(myEI%getAtomicNumber('C')),2.0,'')
       ASSERT_EQ(testArray(myEI%getAtomicNumber('H')),6.0,'')
       ASSERT_EQ(testArray(myEI%getAtomicNumber('O')),1.0,'')
 
       ! Test UF4
       testWord = 'UF4'
-      testArray = mySpec%getElementArray(testWord)
+      testArray = getElementArray(testWord)
       ASSERT_EQ(testArray(myEI%getAtomicNumber('U')),1.0,'')
       ASSERT_EQ(testArray(myEI%getAtomicNumber('F')),4.0,'')
 
       ! Test NaCl
       testWord = 'NaCl'
-      testArray = mySpec%getElementArray(testWord)
+      testArray = getElementArray(testWord)
       ASSERT_EQ(testArray(myEI%getAtomicNumber('Na')),1.0,'')
       ASSERT_EQ(testArray(myEI%getAtomicNumber('Cl')),1.0,'')
 
       ! Test PuO2
       testWord = 'PuO2'
-      testArray = mySpec%getElementArray(testWord)
+      testArray = getElementArray(testWord)
       ASSERT_EQ(testArray(myEI%getAtomicNumber('Pu')),1.0,'')
       ASSERT_EQ(testArray(myEI%getAtomicNumber('O')),2.0,'')
```

Comment:
Should really fill in the messages.  As an example, for the last one I would do something like `'getElementArray('PuO2'), O2'` or something to show what call was being tested and which return value is being checked by that line.

Reply:
I filled in the mesages